pkg/config/output.go

```diff
@@ -486,6 +485,29 @@ func redactEncodedOutputs(out interface {
 	}
 }
 
+func redactStreamKeys(stream *livekit.StreamOutput) {
+	for i, url := range stream.Urls {
+		redacted, ok := redactStreamKey(url)
+		if ok {
+			stream.Urls[i] = redacted
+		}
+	}
+}
+
+var (
+	rtmpRegexp = regexp.MustCompile("^(rtmps?:\\/\\/)(.*\\/)(.*\\/)(\\S*)( live=1)?$")
+)
+
+func redactStreamKey(url string) (string, bool) {
+	match := rtmpRegexp.FindStringSubmatch(url)
+	if len(match) != 6 {
+		return url, false
+	}
+
+	match[4] = redact(match[4])
```

Comment:
On the ingress side, I've seen folks skipping the "application" part of the RTMP url, ie using 

`rtmp://<server>/<stream_key>`

OBS supports sending such content at least. I'm not sure if any service would support such a URL (beyond our ingress...), but it looks like we would not redact it?

Reply:
rtmp2sink requires it

Reply:
AFAICT the stream providers always include the application part as well (I checked twitch, youtube, and facebook). When I switched from rtmpsink to rtmp2sink, I was forced to change the local test streams from `rtmp://localhost:1935/stream1` to `rtmp://localhost:1935/live/stream1`